### PR TITLE
Add missing symbols for our packages

### PR DESCRIPTION
**Pull request by dougbu**
- #10495 and #10508

See second attempt of https://dnceng.visualstudio.com/internal/_releaseProgress?_a=release-pipeline-progress&releaseId=8097 for validation of the symbol packages

### CODE REVIEW

File: src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj [hunk 1]
```diff
@@ -10,7 +10,6 @@
     <PackageType>AzureSiteExtension</PackageType>
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <IncludeSymbols>false</IncludeSymbols>
```
Comment (jkotalik) [inline src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: I'm not sure if this is the right approach. There isn't any code in this package (see https://github.com/aspnet/AspNetCore/tree/master/src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension). What error are you hitting with this package?

Comment (dougbu) [reply src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: This is one of the few packages containing Microsoft.Web.Xdt.Extensions.dll. How else would you suggest we get its .pdb deployed?

Comment (natemcmaster) [reply src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: Maybe I missed it, but I looked at the artifacts in this PR and didn't see Microsoft.Web.Xdt.Extensions.pdb.

**Comment by dougbu** [comment]
Closing in favour of #10641 Will deal with #10495 early in Preview 7.

**Pull request closed without merging**